docs/source/developer/tests.rst

```diff
@@ -42,8 +42,18 @@ directly from Microsoft for any hypervisor.
 Decorating tests
 ~~~~~~~~~~~~~~~~
 
-The ``functional`` decorator marks tests that don't require any pre-existing
-AWS context, and can therefore be run cleanly against live AWS.
+The ``terraform`` decorator marks tests that don't require any pre-existing context and can be run cleanly against live AWS accounts.
```

Comment:
this is a little too backwards looking without explaining how to use the terraform decorator, the example below is a better worked example. i would just remove this section or move the section on using the terraform decorator up.